src/Bridge/Symfony/Bundle/Resources/config/validator.xml

```diff
@@ -9,7 +9,7 @@
             <argument type="service" id="validator" />
             <argument type="service" id="service_container" />
         </service>
-        <service id="ApiPlatform\Core\Validator\ValidatorInterface" alias="api_platform.validator" />
+        <service id="ApiPlatform\Core\Validator\ConstraintValidatorInterface" alias="api_platform.validator" />
```

Comment:
This one is a BC break. If a user is autowiring the `ValidatorInterface`, it will not receive the `api_platform.validator` anymore.

Reply:
Yes, indeed. Sorry, I'm not at ease with BC handling...
May I add another alias instead?
```diff
  <service id="ApiPlatform\Core\Validator\ValidatorInterface" alias="api_platform.validator" />
+ <service id="ApiPlatform\Core\Validator\ConstraintValidatorInterface" alias="api_platform.validator" />
```

Reply:
Yes you can do that :slightly_smiling_face: